brands/amenity/restaurant.json

```
@@ -2554,13 +2554,18 @@
     }
   },
   "amenity/restaurant|Евразия~(Ukraine)": {
+    "matchNames": ["Євразія"],
     "countryCodes": ["ua"],
     "tags": {
       "amenity": "restaurant",
-      "brand": "Евразия",
+      "brand": "Євразія",
+      "brand:ru": "Евразия",
+      "brand:ua": "Євразія",
       "brand:wikidata": "Q66088028",
-      "cuisine": "international",
-      "name": "Евразия"
+      "cuisine": "western,japanese",
```

OSM uses semicolon-delimited lists for `cuisine` tag..
```suggestion
      "cuisine": "western;japanese",
```